sections/admin/clients/new-client.jsx

```diff
@@ -16,8 +17,33 @@ const AdminNewClient = () => {
 
   const handleSubmit = (event) => {
     event.preventDefault();
-    toast.success("Успішно");
-    setTimeout(() => Router.push("/admin/clients"), 500);
+    const dataNewClient = { name, carBrand, phoneNumber, carModel, email, licensePlate };
+
+    const pattern = /^[a-z]+\s[a-z]+$/i;
+
+    if (pattern.test(name)) {
+      createClient(dataNewClient)
+        .then((resp) => {
+          if (resp.status === 201) {
+            toast.success("Клієнт успішно створений!");
+            setTimeout(() => Router.push("/admin/clients"), 500);
```

Comment:
Let`s remove this redirect

Reply:
I didn't add it, it was before me, but okay.